R/modified_update.R

```diff
@@ -446,9 +454,9 @@ mu_scale <- function(name, d) {
   coverage_new <- d$coverage_new
   target_pop_avg <- d$target_pop_estimated_avg
 
-  i <- is_blank(coverage_old) &
```

Comment:
Why is this disabled now?  This was necessary in the past - has something changed to make it no longer needed?

It might be something worth adding as an additional option (`filter_blank_coverage` perhaps?)

Reply:
This is disabled because method 2 calculate updated impact by fvps_new * XXX_rate_tot throughout.

Reply:
So does this break all reports that would use method 1 then?

Reply:
Currently jenner is method 2 only. It is not applied to reports that use method 1. So any report uses method 1 is safe. 

We can make jenner capable of running method 1 at the same time. But, Tini believes that method 1 that Gavi and Gates have been using is not the one we are going to keep.

Reply:
OK - can you remove the code rather than commenting it out then?

Reply:
Done.